packages/app/src/systems/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,110 @@
+import { cssObj } from '@fuel-ui/css';
+import { Icon, Box, Avatar, Flex, Drawer } from '@fuel-ui/react';
+
+import { useAccount } from '../Account';
+import type { Network } from '../Network';
+import { NetworkScreen, useNetworks } from '../Network';
+
+import type { MenuItemObj } from './components';
+import { Menu, NetworkSelector } from './components';
+import { sidebarItems } from './constants';
+
+export function Sidebar() {
+  const { account } = useAccount();
+
+  const { networks, selectedNetwork } = useNetworks({
+    type: NetworkScreen.list,
+  });
+  return (
+    <Flex css={styles.wrapper}>
+      <Flex css={styles.column}>
+        <Flex
+          css={{
+            ...styles.topBorder,
+            ...styles.separator,
+          }}
+        >
+          <Flex css={styles.accountDropdownWrapper}>
+            <Box css={styles.avatarWrapper}>
+              <Avatar.Generated size={'sm'} hash={account?.address as string} />
+            </Box>
+
+            <Icon icon="CaretDown" size={18}></Icon>
+          </Flex>
+          <Drawer.CloseButton
+            css={{ position: 'unset' }}
+            data-testid="drawer_closeButton"
+            aria-label="drawer_closeButton"
+          />
+        </Flex>
+
+        <Menu items={sidebarItems as MenuItemObj[]} />
+      </Flex>
+      <Flex
+        css={{
+          ...styles.bottomBorder,
+          ...styles.separator,
+        }}
+      >
+        <NetworkSelector
+          selected={selectedNetwork as Network}
```

Comment:
```suggestion
          selected={selectedNetwork}
```

Reply:
actually that type cannot be inferred automatically beacuse the selectedNetwork can be undefined also so i just added this as Operator